src/Donations/Micros/PlanetCashSelector.tsx

```diff
@@ -114,13 +114,19 @@ const PlanetCashSelector: FC = (props) => {
     });
   };
 
+  const CountryData = getCountryDataBy("countryCode", country);
```

Comment:
NIT: follow camel case naming convention for variables i.e. `countryData` instead of `CountryData`